ui/src/scenes/Result/index.tsx

```diff
@@ -181,6 +189,7 @@ const Result = () => {
       </Menu>
 
       <Content>
+        <virtual-scroll>tessst</virtual-scroll>
```

Comment:
This looks incomplete.
The purpose of this feature is to virtualize scrolling of items inside `scenes/Schema`